packages/in_app_purchase/pubspec.yaml

```diff
@@ -1,7 +1,7 @@
 name: in_app_purchase
 description: A Flutter plugin for in-app purchases. Exposes APIs for making in-app purchases through the App Store and Google Play.
 homepage: https://github.com/flutter/plugins/tree/master/packages/in_app_purchase
-version: 0.4.1
+version: 0.5.0
```

Comment:
This is not a breaking change and because it is a prerelease, we should only bump the `z` in x.y.z